parcels/field.py

```diff
@@ -34,7 +34,6 @@
 
 
 def _isParticle(key):
-    # TODO: ideally, we'd like to use isinstance(key, BaseParticleAssessor) here, but that results in cyclic imports between Field and ParticleSet.
     if hasattr(key, '_next_dt'):
```

Comment:
well, the comment is not entirely true. The `isinstance(...)` function is nice, but I think one would need to test for `BaseParticleAccessor` or `_Particle`.

Yes, there are more-ideal ways to do that which we cannot because of recursive imports. I don't think though we can resolve that either way.